deploy/roles/log-courier/tasks/main.yml

```diff
@@ -12,12 +12,12 @@
 
 - name: Install log-courier
   yum:
-    name: "{{ item }}"
+    name: [
```

Comment:
Normally I find using the `- item` syntax easier to read, was this change required for some reason?

Reply:
Yes: Ansible gives a deprecation warning, and says the `item` syntax will not be supported in Ansible 2.10.  I took Daniel's advice to make it more readable by putting each package in the list on its own line.